Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:lts-alpine3.12
+
+ADD ./src .
+
+ENTRYPOINT [ "node", "src/app.js" ]
```

Comment:
You should create a working directory to put your code in. 

`WORKDIR /use/src/app`

Then copy over package.json and package-lock.json

`COPY package*.json ./`

Then install dependencies

`RUN npm install`

If you're running in prod you should do `npm ci` but we can talk about why later. 

Then copy over source code

`COPY . .`

Expose your port

`EXPOSE 5000`

Finally your start command

`CMD ["node", "src/app.is"]`

Reply:
Done